packages/gatsby-plugin-image/README.md

```diff
@@ -26,331 +32,225 @@ If you're using the new `GatsbyImage` in addition to `StaticImage`, you'll also
+     return <StaticImage src="../images/dino.png" alt="A dinosaur" />
+   }
+   ```
 
-({ logo }) => <Img src={logo}>
-```
+   If you are using a remote image, pass the image URL in the `src` prop:
 
-...and nor does this:
+   ```jsx
+   import { StaticImage } from "gatsby-plugin-image"
 
-```js
-// ⚠️ Doesn't work
+   export function Kitten() {
+     return <StaticImage src="https://placekitten.com/800/600" alt="A kitten" />
+   }
+   ```
 
-() => {
-    const width = getTheWidthFromSomewhere();
-    return <Img src="trex.png" width={width}>
-}
-```
+   When you build your site, the `StaticImage` component will load the image from your filesystem or from the remote URL, and it will generate all the sizes and formats that you need to support a responsive image.
 
-You can use variables and expressions if they're in the scope of the file, e.g.:
+   Because the image is loaded at build time, you cannot pass the filename in as a prop, or otherwise generate it outside of the component. It should either be a static string, or a local variable in the component's scope.
 
-```js
-//OK
-() => {
-    const width = 300
-    return <Img src="trex.png" width={width}>
-}
-```
+   **Important:** Remote images are downloaded and resized at build time. If the image is changed on the other server, it will not be updated on your site until you rebuild.
 
-```js
-//Also OK
+3. **Configure the image.**
 
-const width = 300
+   You configure the image by passing props to the `<StaticImage />` component. You can change the size and layout, as well as settings such as the type of placeholder used when lazy loading. There are also advanced image processing options available. You can find the full list of options [in the API docs](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image).
 
-() => {
-    const height = width * 16 / 9
-    return <Img src="trex.png" width={width} height={height}>
-}
-```
-
-### API
-
-The only required prop is `src`. The default type is `constrained`. The other props match those of [the new GatsbyImage component](#gatsbyimage). You can also pass in options which are forwarded to [`gatsbyImageData`](#graphql-resolver).
-
-## GatsbyImage
-
-Speedy, optimized images without the work.
-
-GatsbyImage is a React component specially designed to give your users a great image experience. It combines speed and best practices.
-
-Note: GatsbyImage is not a drop-in replacement for `<img>`. It's optimized for fixed width/height images and images that stretch the full-width of a container.
-
-## Table of Contents
-
-- [Problem](#problem)
-- [Solution](#solution)
-- [How to use](#how-to-use)
-- [Types of Responsive Images](#three-types-of-responsive-images)
-- [Gatsby Image Props](#gatsby-plugin-image-props)
+   This component renders a 200px by 200px image of a dinosaur. Before loading it will have a blurred, low-resolution placeholder. It uses the `"fixed"` layout, which means the image does not resize with its container.
```

Comment:
I might move this underneath the example. Because we import Dino in the example two above, I expected it was talking about the defaults applied there.